src/main/scala/jigg/util/XMLUtil.scala

```diff
@@ -134,4 +134,29 @@ object XMLUtil {
       n = elem.key -> elem.value.toString
     } yield n
     ).toSeq
+
+  /** If a string formatted by PrettyPrinter is inputted, the XML.Node.child method
+    * which is used in a subsequent step, e.g. SentenceAnnotator, returns an array
+    * containing empty elements.
+    * To avoid this issue, we create a new XML object using by the XMLUtil.getChildNode method.
+    */
+  def unFormattedXML(node: Node): Node = {
+    val childNode = getNonEmptyChild(node)
+    def recurse(nodes: NodeSeq): NodeSeq = nodes map { n =>
+      val text = XMLUtil.text(n)
+      // To maintain the original text, we define an new node.
+      val newNode = text match{
+        case "" => <xml></xml>
+        case t => <xml>{t}</xml>
+      }
+      val children = n match {
+        case n if getNonEmptyChild(n).filterNot(x => hasChild(x)).length == 0 =>
```

Comment:
This statement may fail when only a part of child elements have non-empty child, for example
```xml
<sentence>
  <tokens>...</tokens>
  <NEs/>
</sentence>
```